features/mogelijkOnjuist.feature

```diff
@@ -315,19 +315,15 @@ Functionaliteit: Mogelijk onjuist
       | Standplaats      | Heeft als nevenadres                     | adresseerbareobjecten | 0301030000017617 | documentdatum       | adressen       | nummeraanduidingIdentificaties | Mogelijk is ten onrechte een nevenadres toegekend of ontbreekt de relatie met een nevenadres. |
       | Ligplaats        | Heeft als nevenadres                     | adresseerbareobjecten | 0301020000024211 | documentdatum       | adressen       | nummeraanduidingIdentificaties | Mogelijk is ten onrechte een nevenadres toegekend of ontbreekt de relatie met een nevenadres. |
 
-    Abstract Scenario: leveren mogelijkOnjuist bij met fields vragen naar link die mogelijk onjuiste identificatie bevat
+    Abstract Scenario: leveren mogelijkOnjuist bij met fields vragen naar link die mogelijk naar een onjuist object verwijst.
       Gegeven in object <Objecttype> is gegeven <Attribuut in de BAG> in onderzoek
       Als ik de resource <Resource> opvraag met fields=_links.<Fields>
       Dan bevat het antwoord property mogelijkOnjuist.<Property> met waarde true
       En bevat het antwoord property mogelijkOnjuist.toelichting met een waarde
```

Comment:
Moet hier de \<Toelichting\> placeholder en bijbehorende toelichtingen niet worden toegevoegd? Dit lijkt een copy\paste van de vorige scenario, maar is de toelichting weggelaten. Of maakt de toelichting niet uit als alleen de link wordt gevraagd? In dat geval kan dan deze stap worden verwijderd omdat het dan niet bijdraagt tot het begrijpen van de gewenste/verwachte functionaliteit